### PR TITLE
Update dependencies for LTS 3.4

### PR DESCRIPTION
**Pull request by notogawa**

### CODE REVIEW

**Comment by Tarrasch** [comment]
Thank you @notogawa for this patch. But what's LTS 3.4?

File: Yesod/Auth/Kerberos.hs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE FlexibleContexts #-}
```
Comment (Tarrasch) [inline Yesod/Auth/Kerberos.hs]: Why is this needed?

**Comment by notogawa** [comment]
LTS is https://github.com/fpco/lts-haskell#readme

**Comment by Tarrasch** [comment]
@notogawa, do you want me to upload a bumped version to hackage as well?

**Comment by asahinet-ohkawa** [comment]
yes, please.

**Comment by notogawa** [comment]
yes, please.

**Comment by Tarrasch** [comment]
https://hackage.haskell.org/package/yesod-auth-kerberos-1.4.3

Enjoy @notogawa!

**Comment by notogawa** [comment]
thanks!
